packages/api/src/controllers/playback.test.ts

```diff
@@ -144,6 +144,37 @@ describe("controllers/playback", () => {
           },
         });
       });
+
+      it("should return playback URL assets from CID", async () => {
+        const cid = "bafyfoobar";
+        await db.asset.update(asset.id, {
+          playbackRecordingId: "mock_recording_id_2",
+          status: {
+            phase: "ready",
+            storage: {
+              ipfs: {
+                data: {
+                  videoFileCid: cid,
+                },
+              },
+            },
+          },
```

Comment:
In this tests it's complaining that a couple of fields are missing for this update
```suggestion
          status: {
            phase: "ready",
            updatedAt: Date.now()
            storage: {
              ipfs: {
                data: {
                  videoFileCid: cid,
                },
                taskIds:{}
              },
            },
          },
```

Reply:
I'll ignore this for now since I believe I've fixed it in the follow-up PR.